framework/view/viewrt/viewer.go

```diff
@@ -0,0 +1,14 @@
+package viewrt
+
+import (
+	"net/http"
+
+	"github.com/livebud/bud/package/router"
+)
+
+type Viewer interface {
+	// TODO: remove handler
+	Handler(route string, props interface{}) http.Handler
+	Render(w http.ResponseWriter, viewPath string, props interface{})
+	Serve(router *router.Router)
+}
```

Comment:
This interface still needs some work, but I just wanted to get something working.